src/content/release/breaking-changes/deep-links-flag-change.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Deep links flag change
+description: >
+  If you use a third party deep linking plugin package for mobile apps,
+  set Flutter's deep linking flag to false.
+---
+
+## Summary
+
+<b>This (potential) breaking change only affects mobile apps and only if you
+use a third party deep linking plugin package.</b>
+
+The default value for Flutter’s deep linking flag has changed from `false` to `true`,
+meaning that deep linking is now opt-in by default.  
+
+## Migration guide
+
+
+If you’re using Flutter’s default deep linking setup, this is not a breaking change for you. 
+
+However, if you’re using a third-party plugin for deep links,
+such as the following, this update introduces a breaking change:
+
+[firebase dynamic links][https://firebase.google.com/docs/dynamic-links]
+[uni_link][https://pub.dev/packages/uni_links]
+[app_links][https://pub.dev/packages/app_links]
```

Comment:
```suggestion
[firebase dynamic links](https://firebase.google.com/docs/dynamic-links)
[uni_link](https://pub.dev/packages/uni_links)
[app_links](https://pub.dev/packages/app_links)
```

Fixing malformed links. @hannah-hyj, if you put the link inline, as have here,  you use curly brackets instead of square brackets.